CMakeLists.txt

```diff
@@ -1051,3 +1051,9 @@ add_subdirectory(${PROJECT_SOURCE_DIR}/render-test)
 if(EXISTS ${PROJECT_SOURCE_DIR}/internal/internal.cmake)
     include(${PROJECT_SOURCE_DIR}/internal/internal.cmake)
 endif()
+
+# installs
+include(GNUInstallDirs)
+install(DIRECTORY include/mbgl
+        DESTINATION "${CMAKE_INSTALL_INCLUDEDIR}"
+        COMPONENT development)
```

Comment:
This install rule will leak to other platforms that are not GNU. Let's confine it to Qt.